flipperdevices/flipperzero-firmware
# [FL-2620] Infrared error message

# What's new

- Show an error message if a remote could not be saved.

# Verification 

1. Remove SD card from Flipper.
2. Fill Flipper's internal storage (e.g. by creating empty directories), leave 8 KB of free space.
3. Go to `Infrared -> Learn New Remote` and try saving a new remote.
4. An error message must be shown. The app should return to the start menu upon closing it.
5. If the remote is saved successfully, make sure that the Internal Storage reports 0 KB free and repeat step 4.

# Checklist (For Reviewer)
